pkg/driver/controller.go

```diff
@@ -185,6 +185,7 @@ func (cs *controller) CreateVolume(
 	var selected string
 
 	volName := req.GetName()
+	pool := req.GetParameters()["poolname"]
```

Comment:
Should upgrade be considered? Is there way to update this value for the older volumes?

Reply:
not needed @kmova . It will not impact any funtionality.